tests/php/test_class.jetpack-affiliate.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
Since we're removing the file, you may have to remove it from `phpunit.xml.dist` too.
https://github.com/Automattic/jetpack/blob/01b62d9b421c3fa7ae97c00f434a4c3343d2d4af/phpunit.xml.dist#L82